src/openeo_grass_gis_driver/actinia_processing/actinia_interface.py

```diff
@@ -130,6 +132,17 @@ def delete_resource(self, resource_id: str) -> Tuple[int, dict]:
 
         return r.status_code, data
 
+    def list_locations(self) -> Tuple[int, dict]:
```

Comment:
is this method called anywhere?

Reply:
Yes in src/openeo_grass_gis_driver/authentication.py line 39